examples/minimal/packages/contracts/worlds.json

```diff
@@ -0,0 +1,9 @@
+{
+  "31337": {
+    "address": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
+  },
+  "4242": {
+    "address": "0xf0F5e9b00b92f3999021fD8B88aC75c351D93fc7",
+    "blockNumber": 10591501
+  }
```

Comment:
another potential structure for this is

```
[{
  chainId: 31337,
  address: "0x...",
  blockNumber: 123456,
}],
```
but I figured that'd be a little more painful to "splice" if we need to replace a world address/block number for a particular chain